### ISSUE TITLE
config suggestion?

### ISSUE THREAD

**Issue by el-tocino** [MycroftAI/mycroft-precise#178]
Instead of just passing the config on the command line, it might be easier to pass a (json?) file with the configuration in it.  In particular, the paths to the relevant wake and test directories, as well as batch size and other items as preferred.

**Comment by MatthewScholefield** [comment]
Yeah, I think this would be great. Perhaps a hybrid approach could work? I'm not sure the best way but it'd be nice if we could support both.
